e2e/test/scenarios/onboarding/setup/setup.cy.spec.ts

```diff
@@ -190,12 +191,27 @@ describe("scenarios > setup", () => {
       );
       cy.button("Finish").click();
 
+      // we need a mocked response to make sure we're not hitting the real endpoint
```

Comment:
This is actually really important, without this it was succeeding locally but failing in ci, i think because in ci the real endpoint fails and the request never replies.
If we instead mock the response, we're not hitting the real endpoint at all